### PR TITLE
Update Dictionary.cpp with some better fitting icons

### PR DESCRIPTION
**Pull request by tioan**

### CODE REVIEW

File: src/Dictionary.cpp [hunk 1]
```diff
@@ -99,12 +99,15 @@ const char HAupClass[] PROGMEM = {"duration"};
...
 const char HAbtnLName[] PROGMEM = {"Button left"};
+const char HAtransIcon[] PROGMEM = {"mdi:button-pointer"};
 
 const char HAbtnMID[] PROGMEM = {"%s_btnM"};
 const char HAbtnMName[] PROGMEM = {"Button select"};
+const char HAtransIcon[] PROGMEM = {"mdi:button-pointer"};
 
 const char HAbtnRID[] PROGMEM = {"%s_btnR"};
 const char HAbtnRName[] PROGMEM = {"Button right"};
+const char HAtransIcon[] PROGMEM = {"mdi:button-pointer"};
```
Comment (wilfredktlr) [inline src/Dictionary.cpp:110]: Maybe these are better alternatives (visually speaking)?
- `mdi:chevron-left-box-outline`
- `mdi:chevron-down-box-outline`  (or `mdi:circle-box-outline`)
- `mdi:chevron-right-box-outline`

File: src/Dictionary.cpp [hunk 2]
```diff
@@ -99,12 +99,15 @@ const char HAupClass[] PROGMEM = {"duration"};
...
 
 const char HAramRID[] PROGMEM = {"%s_ram"};
 const char HAramIcon[] PROGMEM = {"mdi:application-cog"};
```
Comment (wilfredktlr) [inline src/Dictionary.cpp:113]: Alternative? `mdi:memory`

Comment (wilfredktlr) [inline src/Dictionary.cpp:113]: Alternative? `mdi:memory`

**Comment by tioan** [comment]
I have blueforcer give in https://discord.com/channels/546407049148366859/1100368501820104704 to create a poll on the 3 options.

original
original

option 1
option1

option 2
option2

option 3
option3

**Comment by wilfredktlr** [comment]
I see there is a bug when [compiling](https://github.com/Blueforcer/awtrix-light/actions/runs/4909437876/jobs/8765841649) this change. The line below has been copied three times but the name was not updated to `HAbtnLIcon`, `HAbtnMIcon` and `HAbtnRIcon` which causes an error (`redefinition of 'const char HAtransIcon []'`).

```
const char HAtransIcon[] PROGMEM = {"mdi:..."};
```

**Pull request merged**